squad/core/models.py

```diff
@@ -494,13 +494,22 @@ class Test(models.Model):
     name = models.CharField(max_length=256, db_index=True)
     result = models.NullBooleanField()
     log = models.TextField(null=True, blank=True)
+    known_issue = models.ForeignKey('KnownIssue', null=True, blank=True)
```

Comment:
which KnownIssue will be selected if there is more than 1 active that matches the test name?

Reply:
good point. this will probably have to be turned into a many to many relationship